### DISCUSSION TITLE
On Demo Website(Boilerplate), Checkout not loading and till the cart everything is fine

### DISCUSSION THREAD

**Discussion by aiopsgroup** [scayle/community#2] [Q&A]
Hello,

I have set up a boilerplate on my local and checkout is not loading. I can see some CORS errors in the console on checkout page and below are the URL.
![image](https://github.com/user-attachments/assets/a32848ba-c867-48cb-ae17-b51808fe729a)
https://fiji-demo.checkout.api.scayle.cloud/api/co/v3/state
https://fiji-demo.checkout.api.scayle.cloud/api/co/v3/state/config
https://checkout-cdn.aboutyou.cloud/shops/fiji-demo/k2/overwrites.css

I believe there is some configuration that I am missing, can you help me with that?

Also, I am facing the same error while trying to initialize the web component of checkout according to below tutorial.
https://scayle.dev/en/developer-guide/checkout/authentication

![image](https://github.com/user-attachments/assets/572fd242-6dec-4407-bf5c-cb0fcae2d7b8)

**Comment by HenriBeck** [comment]
Hey,

this is a configuration issue in the Panel related to CORS. Please ask this question through the official tech support from SCAYLE.
